Tasks/PackerBuildV0/DefaultTemplates/custom.managed.linux.template.json

```diff
@@ -0,0 +1,68 @@
+{
+  "variables": {
+    "client_id": "{{env `client_id`}}",
+    "client_secret": "{{env `client_secret`}}",
+    "subscription_id": "{{env `subscription_id`}}",
+    "tenant_id": "{{env `tenant_id`}}",
+    "object_id": "{{env `object_id`}}",
+    "resource_group": "{{env `resource_group`}}",
+    "storage_account": "{{env `storage_account`}}",
+    "image_url": "{{env `image_url`}}",
```

Comment:
Should we also enable taking source managed image as input? one customer was asking for that.

Reply:
As mentioned above I have tested with my own custom managed and unmanaged templates which have dependencies on other files and folders in the repo. Can test source managed image as a part of a separate PR.

Reply:
I believe it is not just testing work. You will need to re-word the input to signify either a VHD URL or a managed image id. Also, a variable called `custom_managed_image_name` should be passed to packer template. Adding @azooinmyluggage to provide correct guidance regarding input changes.
I agree with taking this work as a separate PR.

Reply:
Totaly Agree, thats what I had in mind.